cogs/commands/reminders.py

```diff
@@ -102,6 +104,53 @@ def add_base(self, reminder):
             logging.exception(e)
             return {"content": f"Something went wrong with the database"}
 
+    @reminder.command()
+    async def list(self, ctx: Context):
+        """
+        Lists all reminders upcoming
+        """
+
+        reminders = (
+            db_session.query(Reminder)
+            .filter(
+                Reminder.trigger_at >= datetime.now(),
+                Reminder.triggered == False,
+            )
```

Comment:
I'd recommend adding a reminder author check here (you need to fetch the DB user with `utils.get_database_user(ctx.author)` then compare to the reminder user)